host/xtest/regression_1000.c

```diff
@@ -2345,3 +2349,142 @@ static void xtest_tee_test_1032(ADBG_Case_t *c)
 }
 ADBG_CASE_DEFINE(regression, 1032, xtest_tee_test_1032,
 		"Register read-only shared memory");
+
+static void xtest_tee_test_1033(ADBG_Case_t *c)
+{
+	TEEC_Result res = TEEC_ERROR_GENERIC;
+	TEEC_Session session = { };
+	uint32_t ret_orig = 0;
+
+	/*
+	 * TA will ping the test plugin during open session operation. 
+	 * Supplicant plugins are optional: warn and nicely exit if not found
```

Comment:
Optional? I can't see that in https://github.com/OP-TEE/optee_client/pull/239

Reply:
Fixed, thanks.